src/Http/TransformsFlexibleErrors.php

```diff
@@ -3,22 +3,21 @@
 namespace Whitecube\NovaFlexibleContent\Http;
 
 use Illuminate\Support\Str;
-use Illuminate\Http\JsonResponse;
 use Symfony\Component\HttpFoundation\Response;
 use Whitecube\NovaFlexibleContent\Flexible;
-use Whitecube\NovaFlexibleContent\Http\FlexibleAttribute;
+use Illuminate\Http\JsonResponse;
 
 trait TransformsFlexibleErrors
 {
     /**
      * Updates given response's errors for the concerned flexible fields
      *
-     * @param  \Illuminate\Http\JsonResponse  $response
-     * @return \Illuminate\Http\JsonResponse
+     * @param  $response
+     * @return \Symfony\Component\HttpFoundation\Response
      */
-    protected function transformFlexibleErrors(JsonResponse $response)
+    protected function transformFlexibleErrors($response)
```

Comment:
Could we keep the `Response` type hint in there like the issue suggested? Is there a reason you left it out?

Reply:
Re-added it

Reply:
Thanks!